espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/Handlers/CreateSession.java

```diff
@@ -20,42 +17,24 @@
 
 public class CreateSession implements RequestHandler {
 
-    public BaseResponse handle(NanoHTTPD.IHTTPSession session, Map<String, String> uriParams) {
+    public BaseResponse handle(NanoHTTPD.IHTTPSession session, Map<String, Object> params) {
         Session appiumSession = new Session();
         AppiumResponse appiumResponse = new AppiumResponse();
```

Comment:
No, this is implementing the interface, since there is no session or anything before this.